requirements/pip-tools.in

```diff
@@ -1,4 +1,5 @@
 # Just the dependencies to run pip-tools, mainly for the "upgrade" make target
 -c constraints.txt
 
-pip-tools                 # Contains pip-compile, used to generate pip requirements files
+pip-tools<6                 # Contains pip-compile, used to generate pip requirements files
+six
```

Comment:
The only thing I see that needs six is `docs/conf.py` but `requirements/doc.in` is not importing 'six'. Should we move this line to doc.in?

Reply:
I'll give that a try, thanks!

Reply:
Moved it to docs.in even though a full `make upgrade` will pull it first into travis.txt as a tox requirement.